frontend/src/components/ChatBox/ChatBox.tsx

```diff
@@ -111,7 +111,7 @@ function ChatBox({
 		}
 		response.defenceReport.alertedDefences.forEach((triggeredDefence) => {
 			// get user-friendly defence name
-			const defenceName = ALL_DEFENCES.find((defence) => {
+			const defenceName = DEFAULT_DEFENCES.find((defence) => {
```

Comment:
Ah, interesting... Is this because System Role and the Q&A LLM are not defences that are "triggered" like the others? If so, this provides useful clarity 👍

Reply:
True they're not "triggered", but then also the message transformation defences aren't triggered either.
It's mostly because those two aren't configurable in level 3